react-components/src/components/RevealToolbar/LayersContainer/elements.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { Checkbox, Menu } from '@cognite/cogs.js';
+import styled from 'styled-components';
+
+export const StyledCheckbox = styled(Checkbox)`
+  margin-left: 10px;
+`;
+
+export const StyledMenu = styled(Menu)`
+  display: flex;
+  flex-direction: row;
+  box-shadow: none;
+  padding: 0px;
```

Comment:
```suggestion
  padding: 6px;
```
Shadow should be visible as it helps distinguish the menu form the background.